app/build.gradle

```diff
@@ -36,19 +36,19 @@ android {
 
 dependencies {
     //noinspection GradleDependency TODO: UI bug
-    implementation "com.google.android.material:material:1.0.0"
+    implementation "com.google.android.material:material:1.3.0-alpha02"
     implementation "androidx.cardview:cardview:1.0.0"
-    implementation "androidx.browser:browser:1.2.0"
-    implementation "androidx.preference:preference:1.1.0"
-    implementation "androidx.recyclerview:recyclerview:1.2.0-alpha01"
-    implementation "androidx.fragment:fragment:1.2.3"
-    implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
+    implementation "androidx.browser:browser:1.3.0-alpha05"
+    implementation "androidx.preference:preference:1.1.1"
+    implementation "androidx.recyclerview:recyclerview:1.2.0-alpha05"
+    implementation "androidx.fragment:fragment:1.3.0-alpha07"
+    implementation 'androidx.constraintlayout:constraintlayout:2.0.0-rc1'
     implementation 'com.afollestad.material-dialogs:commons:0.9.0.2'
     implementation 'com.github.mtotschnig:StickyListHeaders:2.7.1'
     implementation "com.github.topjohnwu.libsu:busybox:2.5.0"
-    implementation 'com.squareup.picasso:picasso:2.5.2'
+    implementation 'com.squareup.picasso:picasso:2.8'
```

Comment:
Picasso 2.8 modified the with(Context) method to get() relative to 2.5.2.

It needs to be modified RepoParser.java:318 at the same time to be compatible with the changes in the new version.